source/sass/patterns/organisms/content-header.scss

```diff
@@ -1,34 +1,106 @@
 @import "../../functions/types";
 @import "../../mixins/scale";
 @import "../../mixins/decorations";
+@import "../../mixins/logical-properties";
 @import "../../mixins/media-query";
 @import "../../mixins/positioning";
 @import "../../mixins/spacing";
 @import "../../settings/baselinegrid";
+@import "../../settings/color";
 @import "../../settings/grid";
 
```

Comment:
Missing @import "../../vendor/sass-mq/_mq.scss"

Reply:
One of the few cases where we don't import the actual file where something is an use

https://github.com/libero/pattern-library/blob/d95e50f2743c90be25ac610099412127b304c89b/source/sass/patterns/organisms/content-header.scss#L5

due to it needing settings.